media-gfx/yafaray/yafaray-3.5.1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+# doesn't build with ninja when qt5 and python USE flags are both enabled
+CMAKE_MAKEFILE_GENERATOR="emake"
+
+inherit cmake flag-o-matic python-single-r1
+
+DESCRIPTION="A free open-source montecarlo raytracing engine"
+# Regular homepage is currently down. Upstream is working on this.
+HOMEPAGE="https://www.yafaray.org https://github.com/YafaRay/libYafaRay"
+SRC_URI="https://github.com/YafaRay/libYafaRay/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+S="${WORKDIR}/libYafaRay-${PV}"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+fastmath +fasttrig jpeg opencv openexr png python qt5 tiff truetype"
+RESTRICT="test"
+
+REQUIRED_USE="
+	python? ( ${PYTHON_REQUIRED_USE} )
+"
+
+# Note: according to upstream, the blender plugin doesn't work with >=blender-2.8 (yet).
+RDEPEND="
+	dev-libs/libxml2:2
+	sys-libs/zlib
+	jpeg? ( virtual/jpeg )
+	opencv? ( media-libs/opencv:= )
+	openexr? ( media-libs/openexr:= )
+	png? ( media-libs/libpng:= )
+	python? ( ${PYTHON_DEPS} )
+	qt5? ( dev-qt/qtwidgets:5 )
+	tiff? ( media-libs/tiff )
+	truetype? ( media-libs/freetype:2 )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="python? ( dev-lang/swig )"
```

Comment:
You might want to add `${PYTHON_DEPS}` here as well, inside `python? ( )`.

Reply:
Is it still needed if it's already in `RDEPEND`?

Reply:
Well I'd imagine python is still needed in the build-system, I see it goes to `DEPEND` but it usually gets put to `BDEPEND`.

Reply:
Ok thanks, I'm gonna fix this together with the cflags issue.

Reply:
(I already did for ::gentoo repo) :)

Reply:
Thanks a lot! :)